### PR TITLE
Add Diar-az

### PR DESCRIPTION
**Pull request by ArnarFreyrKristinsson**
Diar-az creates files for a (diarization) corpus from Gecko and provides organization, cleaning and correction of data for Kaldi to Gecko to Kaldi/corpus and back.

### CODE REVIEW

**Comment by wq2012** [comment]
I think this should fall into "Other software" instead of "Diarization dataset".

This is not a new dataset. It's just a format conversion tool, is it correct?

**Comment by judyfong** [comment]
Its a tool specifically for the ruv-di dataset

**Comment by wq2012** [comment]
If so, we should add ruv as a dataset, and this repo as "Other Software".

**Comment by judyfong** [comment]
The dataset was never published, only the resulting models. Also, yes that
dataset should be added but it was also lost in a cyber security attack in
January 2024 on Reykjavik University’s servers. If you want, you could put
a placeholder text for the RÚV-DI dataset here in this repo and we could
try to recreate the dataset. We have a license that lists all the shows and
episodes contained within the dataset. So we could recreate it from that.

Other software works in my opinion.

**Comment by ArnarFreyrKristinsson** [comment]
Yes, I think Other software works and maybe a better fit, as it's not really a dataset, rather it was a tool to support the ruv-di dataset. To correct this, should this pull request be just updated or a new one created?

**Comment by wq2012** [comment]
> To correct this, should this pull request be just updated or a new one created?

I'm OK either way.

**Comment by ArnarFreyrKristinsson** [comment]
> > To correct this, should this pull request be just updated or a new one created?
> 
> I'm OK either way.

Fixed, added to other software.

**Comment by judyfong** [comment]
@afk0901 i believe you also need to put the placeholder text for the dataset for this pr to be properly closed. 

In terms of recreating the dataset i believe it's actually best if @wq2012 recreates the dataset with daan and pet of google. And @afk0901 finish our writeup of this dataset creation. When we are both done we compare notes on arxiv and write the dataset paper together for interspeech, icassp, or sand2025, or wand in october.

**Comment by judyfong** [comment]
For continuity and clarity I believe it's best if my second paragraph is dealt with separately, not in this pr. Thus i have created a new issue for it within this repo.

**Comment by wq2012** [comment]
> > > To correct this, should this pull request be just updated or a new one created?
> > 
> > 
> > I'm OK either way.
> 
> Fixed, added to other software.

I didn't see the change.

File: README.md [hunk 1]
```diff
@@ -295,6 +296,7 @@ Team in the Inaugural DIHARD Challenge](https://www.isca-speech.org/archive/pdfs
 | [VoxConverse](https://github.com/joonson/voxconverse) | TBD | TBD | Free | VoxConverse is an audio-visual diarisation dataset consisting of over 50 hours of multispeaker clips of human speech, extracted from YouTube videos |
 | [MiniVox Benchmark](https://github.com/doerlbh/MiniVox) | [MiniVox Benchmark](https://github.com/doerlbh/MiniVox) | en | Free | MiniVox is an automatic framework to transform any speaker-labelled dataset into continuous speech datastream with episodically revealed label feedbacks. |
 | [The AliMeeting Corpus](https://github.com/yufan-aslp/AliMeeting) | Together with audios | zh | Free |  |
+| RÚV-DI dataset | TBD | is | TBD | |
```
Comment (wq2012) [inline README.md]: please remove this

Comment (ArnarFreyrKristinsson) [reply README.md]: Removed.

**Pull request merged**